src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -2191,28 +2191,14 @@ internal static void LogScriptBlockEnd(ScriptBlock scriptBlock, Guid runspaceId)
         internal bool HasCleanBlock { get => AstInternal.Body.CleanBlock != null; }
     }
 
-    [Serializable]
-    internal class ScriptBlockSerializationHelper : ISerializable, IObjectReference
+    internal class ScriptBlockSerializationHelper
```

Comment:
This is an internal class, so if it's not used by anything, I think it can be removed.

Reply:
I see it is used in 1 place. I will keep it.

Reply:
It's used in `ScriptBlock.GetObjectData(SerializationInfo info, StreamingContext context)`, which should be removed, isn't it?

Reply:
It's used in `ScriptBlock.GetObjectData(SerializationInfo info, StreamingContext context)`, which should be removed, isn't it?

Reply:
Removed both.